README.md

```diff
@@ -1,18 +1,29 @@
-=== AMP ===
-Contributors: google, xwp, automattic, westonruter, albertomedina, schlessera, swissspidy, pierlo, johnwatkins0, joshuawold, ryankienstra
-Tags: amp, mobile, optimization, accelerated mobile pages, framework, components, blocks, performance, ux, seo, official
-Requires at least: 4.9
-Tested up to: 5.6
-Stable tag: 2.0.10
-License: GPLv2 or later
-License URI: http://www.gnu.org/licenses/gpl-2.0.html
-Requires PHP: 5.6
+# AMP
 
-The Official AMP Plugin, supported by the AMP team. Formerly Accelerated Mobile Pages, AMP enables great experiences across both mobile and desktop.
+Contributors: google, xwp, automattic, westonruter, albertomedina, schlessera, swissspidy, pierlo,   johnwatkins0, joshuawold, ryankienstra
+Tags: amp, mobile, optimization, accelerated mobile pages, framework, components, blocks, performance,   ux, seo, official  
+Requires at least: 4.9  
+Tested up to: 5.6  
+Stable tag: 2.0.10  
+License: GPLv2 or later  
+License URI: http://www.gnu.org/licenses/gpl-2.0.html  
+Requires PHP: 5.6  
 
-== Description ==
+![The Official AMP Plugin for WordPress](.wordpress-org/banner-1544x500.png)
 
-This official plugin from the AMP project enables AMP content publishing with WordPress in a way that is fully and seamlessly integrated with the standard mechanisms of the platform. The key features are the following:
+The official AMP Plugin, supported by the AMP team. Formerly Accelerated Mobile Pages, AMP enables great experiences across both mobile and desktop.
+
+[![Build Status](https://github.com/ampproject/amp-wp/workflows/Build,%20test%20&%20measure/badge.svg)](https://github.com/ampproject/amp-wp/actions?query=branch%3Adevelop+workflow%3A%22Build%2C+test+%26+measure%22) [![Coverage Status](https://img.shields.io/codecov/c/github/ampproject/amp-wp/develop.svg)](https://codecov.io/gh/ampproject/amp-wp) [![Built with Grunt](https://gruntjs.com/cdn/builtwith.svg)](http://gruntjs.com)
```

Comment:
Putting each badge on its own line should help with overall maintenance of the readme (and doesn't change how it's rendered).

```suggestion
[![Build Status](https://github.com/ampproject/amp-wp/workflows/Build,%20test%20&%20measure/badge.svg)](https://github.com/ampproject/amp-wp/actions?query=branch%3Adevelop+workflow%3A%22Build%2C+test+%26+measure%22)
[![Coverage Status](https://img.shields.io/codecov/c/github/ampproject/amp-wp/develop.svg)](https://codecov.io/gh/ampproject/amp-wp)
[![Built with Grunt](https://gruntjs.com/cdn/builtwith.svg)](http://gruntjs.com)
```

Reply:
Good point.